src/components/AccessRequest/FormDialog.tsx

```diff
@@ -88,43 +87,47 @@ const FormDialog: FC<Props> = ({ onClose, ...rest }) => {
   const { handleSubmit, register, control, formState } = useForm<InputForm>({
     defaultValues: {
       role: RoleOptions.includes(user.role) ? user.role : "Submitter",
-      organization: user?.organization?.orgName || "",
+      studies: [],
+      additionalInfo: "",
     },
   });
   const { errors, isSubmitting } = formState;
 
-  const { data } = useQuery<ListOrgNamesResp>(LIST_ORG_NAMES, {
-    context: { clientName: "backend" },
-    fetchPolicy: "cache-first",
-    onError: () => {
-      enqueueSnackbar("Unable to retrieve organization list.", {
-        variant: "error",
-      });
-    },
-  });
+  const { data } = useQuery<ListApprovedStudiesResp, ListApprovedStudiesInput>(
+    LIST_APPROVED_STUDIES,
+    {
+      context: { clientName: "backend" },
+      fetchPolicy: "cache-first",
+      onError: () => {
+        enqueueSnackbar("Unable to retrieve approved studies list.", {
+          variant: "error",
+        });
+      },
+    }
+  );
 
   const [requestAccess] = useMutation<RequestAccessResp, RequestAccessInput>(REQUEST_ACCESS, {
     context: { clientName: "backend" },
     fetchPolicy: "no-cache",
   });
 
-  const sortedOrgs = useMemo<string[]>(
+  const sortedStudies = useMemo<ApprovedStudy[]>(
```

Comment:
This works fine, but should we use the API sorting instead?

Reply:
Agreed, switched to API sorting in [3171a0f](https://github.com/CBIIT/crdc-datahub-ui/pull/553/commits/3171a0f82a5ee4740d524469d8515efa0eb1658c) and [d20a75f](https://github.com/CBIIT/crdc-datahub-ui/pull/553/commits/d20a75f472d61bae655c91b4fb412bd6d19fd29a).